app/test_split_integer.py

```diff
@@ -2,20 +2,34 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    goal = split_integer(6, 2)
+    assert (
+        sum(goal) == 6
+    )
```

Comment:
Do not overcomplex the structure and do not allocate new variables, all tests should be one-liners and contain one `assert` statement. Make sure that you compare the function result to `list` with values where needed for simplicity, without any additional modifications